java/org/contikios/cooja/mspmote/interfaces/MspMoteID.java

```diff
@@ -154,9 +154,8 @@ public void removed() {
 	private void addMonitor(String variable, MemoryMonitor monitor) {
 	    if (moteMem.variableExists(variable)) {
 	        int address = (int) moteMem.getVariableAddress(variable);
-	        if ((address & 1) != 0) {
-	            // Variable can not be a word - must be a byte
-	        } else {
+          // Variable can not be a word - must be a byte
```

Comment:
This comment becomes strange when moved here as the check ensures the variable is not a byte on an odd address.

Reply:
Fixed.